clusterloader2/testing/dra/config.yaml

```diff
@@ -63,10 +63,30 @@ steps:
       Params:
         action: start
         labelSelector: job-type = long-running
+    - Identifier: FastFillClaimAllocationLatency
+      Method: ResourceClaimAllocationLatency
+      Params:
+        action: start
     - Identifier: FastFillSchedulingMetrics
       Method: PrometheusSchedulingMetrics
       Params:
         action: start
+    - Identifier: FastFillDRAMetrics
+      Method: GenericPrometheusQuery
+      Params:
+        action: start
+        metricName: fastfill_kubelet_latencies
+        metricVersion: v1
+        unit: s
+        queries:
+          - name: p99_dra_prepare_resources
+            query: histogram_quantile(0.99, sum(rate(dra_operations_duration_seconds_bucket{operation_name="PrepareResources"}[%v])) by (le))
+          - name: p99_dra_unprepare_operations
+            query: histogram_quantile(0.99, sum(rate(dra_operations_duration_seconds_bucket{operation_name="UnprepareResources"}[%v])) by (le))
+          - name: p99_dra_grpc_node_prepare_resources
+            query: histogram_quantile(0.99, sum(rate(dra_grpc_operations_duration_seconds_bucket{method_name="NodePrepareResources"}[%v])) by (le))
```

Comment:
It's not clear from [the documentation](https://github.com/kubernetes/kubernetes/blob/2bdcad7419c7a584da4174513e36f867e88424df/pkg/kubelet/metrics/metrics.go#L1010-L1032) (cc @bart0sh), but "method_name", in contrast to "operation_name", includes the full gRPC method name. Currently that is `/k8s.io.kubelet.pkg.apis.dra.v1beta1.DRAPlugin/NodePrepareResources` and `/k8s.io.kubelet.pkg.apis.dra.v1beta1.DRAPlugin/NodeUnprepareResources`.

@bart0sh: is that correct? Do we have any tests for these? Can you update the docs?

Reply:
Below you are using `=~".` but not here.